llvm/test/MC/Disassembler/AMDGPU/gfx11_dasm_ds.txt

```diff
@@ -4447,76 +4447,76 @@
 # GFX11: ds_store_b128 v255, v[2:5] offset:65535 ; encoding: [0xff,0xff,0x7c,0xdb,0xff,0x02,0x00,0x00]
 0xff,0xff,0x7c,0xdb,0xff,0x02,0x00,0x00
 
-# GFX11: ds_store_b16 v0, v1                     ; encoding: [0x00,0x00,0x7c,0xd8,0x00,0x01,0x00,0x00]
+# GFX11: ds_store_b16 v0, v1.l                   ; encoding: [0x00,0x00,0x7c,0xd8,0x00,0x01,0x00,0x00]
```

Comment:
Using the `.l` syntax is not right here. This is not a true16 operand, cos you can't specify v0.h but you can specify v255.l (with an 8-bit instruction field). So it's a normal 32-bit operand but the instruction happens to only use the low 16 bits of data from it.